android-sdk/versions.gradle

```diff
@@ -28,7 +28,7 @@ versions.network = network_versions
 
 // JNI
 def jni_versions = [:]
-jni_versions.mobilecoin = '3.0.0-pre6'
+jni_versions.mobilecoin = '2.0.1-pre0'
```

Comment:
Should this be 4.0.0.1? `versioned_crypto_box_encrypt` [was added in 4.0.0.1](https://github.com/mobilecoinofficial/android-bindings/pull/5/files)

Reply:
Yes, the version change is coming in the next PR for the release update. I just left this here to avoid the conflict in the next PR

Reply:
Why would there be a conflict? This is a JNI dependency, the code in this PR won't work without it.